css/style.css

```diff
@@ -0,0 +1,3958 @@
+  box-shadow: var(--box-shadow-m);
+  background-color: var(--secondary-color);
+}
+
+.values-card-icon {
+  display: -webkit-inline-box;
+  display: -ms-inline-flexbox;
+  display: inline-flex;
+  max-width: 56px;
+  padding: 12px;
+  background-color: var(--accent-color);
+  border-radius: var(--border-radius-s);
+}
+
+.values-card-content :first-child {
+  margin-bottom: 4px;
+}
+
+.card-title-overflow {
+  display: -webkit-box;
+  -webkit-box-orient: vertical;
+  -webkit-line-clamp: 1;
+  overflow: hidden;
+}
+
+.values-card-content :last-child {
+  color: var(--opacity-80);
+}
+
+/**************** Products section *****************/
+
+.products {
+  text-align: center;
+}
+
+.products-images {
+  margin: 0 auto;
+  margin-top: 48px;
+  padding-top: 48px;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  overflow-x: auto;
+  white-space: nowrap;
+  scrollbar-width: none;
+}
+
+.products-images-item {
+  display: block;
+  border-radius: var(--border-radius-s);
+  cursor: pointer;
+  -webkit-transition: -webkit-transform var(--trasition-effect);
+  transition: -webkit-transform var(--trasition-effect);
+  -o-transition: transform var(--trasition-effect);
+  transition: transform var(--trasition-effect);
+  transition: transform var(--trasition-effect),
+    -webkit-transform var(--trasition-effect);
+}
+
+.products-images-item:hover {
+  -webkit-transform: translateY(-48px);
+  -ms-transform: translateY(-48px);
+  transform: translateY(-48px);
+}
+
+/************* Awards section ****************/
+
+.awards {
+  padding-block: 100px;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-orient: horizontal;
+  -webkit-box-direction: reverse;
+  -ms-flex-direction: row-reverse;
+  flex-direction: row-reverse;
+  -webkit-column-gap: 80px;
+  -moz-column-gap: 80px;
+  column-gap: 80px;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  position: relative;
+}
+
+.awards::after {
+  content: "";
+  width: 550px;
+  height: 430px;
+  position: absolute;
+  top: 0;
+  left: 20px;
+  z-index: -1;
+  border-radius: var(--border-radius-m);
+  background-image: url("../img/img-backgounds/awards-bg.webp");
+  background-color: var(--bg-color);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.awards-content {
+  max-width: 466px;
+}
+
+.awards-card-list {
+  max-width: 370px;
+  width: 100%;
+  display: -ms-grid;
+  display: grid;
+  row-gap: 10px;
+  position: relative;
+  bottom: -95px;
+}
+
+.awards-card {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-column-gap: 16px;
+  -moz-column-gap: 16px;
+  column-gap: 16px;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  padding: 16px;
+  -webkit-box-shadow: var(--box-shadow-m);
+  box-shadow: var(--box-shadow-m);
+  background-color: var(--secondary-color);
+  border-radius: var(--border-radius-s);
+}
+
+.awards-card::before {
```

Comment:
Looks more like it's part of the content and not a decorative element, use `img` instead

Reply:
#Fixed